requirements/requirements.in

```diff
@@ -45,3 +45,4 @@ django-field-audit>=1.2.7
 turn-python>=0.2.0
 jinja2
 django-taggit
+transformers
```

Comment:
What do we need this for?

Reply:
I got this:

> Could not import transformers python package. This is needed in order to calculate get_token_ids. Please install it with `pip install transformers`.

that originated from [here](https://github.com/langchain-ai/langchain/blob/ad3fd44a7f83e3c5e6b25024ce1b49f1a828defe/libs/core/langchain_core/language_models/base.py#L43).

Reply:
👍🏻 could you add a comment